source/patterns/01-molecules/teaser/teaser.twig

```diff
@@ -37,7 +37,7 @@
             {%- block details -%}
 
               {%- if text is defined -%}
-                {%- include 'atoms-paragraph' -%}
+                {%- include 'atoms-paragraph' with {text: text, attributes: {class: ['teaser__details']}} -%}
```

Comment:
Should this allow for augmenting pre-existing `attributes`? Does this need `only` to restrict the scope?

Reply:
Doesn't need `only` as the `with` already restricts it. `text: text` is redundant (without `only`). `attributes` should allow for pre-existing attributes/classes, as the block in the template could be extended to add one.

Reply:
Addressed in 3b445d2.